_data/pixels.json

```
@@ -2,7 +2,7 @@
   "data": [
     { "y": 1, "x": 3, "color": "#babaca", "username": "andrescalco" },
     { "y": 1, "x": 4, "color": "#006699", "username": "adrianolsk" },
-    { "y": 1, "x": 5, "color": "#F22F46", "username": "<UNCLAIMED>" },
+    { "y": 1, "x": 5, "color": "#F22F46", "username": "venoox" },
```

Should be like this:

```suggestion
    { "y": 1, "x": 5, "color": "#F22F46", "username": "Venoox" },
```